src/ConnectedMode/BranchMatcher.cs

```diff
@@ -142,7 +142,7 @@ private int GetDistance(Commit[] headCommits, Branch branch, int closestDistance
                 CodeMarkers.Instance.GetDistanceStart(branch.FriendlyName);
                 for (int i = 0; i < headCommits.Length; i++)
                 {
-                    if (i >= closestDistance) { break; }
+                    if (i > closestDistance) { break; }
```

Comment:
needs to iterate one more so we can find equal distance too. Before we ignored it because we discarded equal distance.